### PR TITLE
Tutorial code

### PR DESCRIPTION
**Pull request by leplatrem**
Code from plugin tutorial in Kinto official docs + package + tests

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,7 @@
+include CHANGELOG.rst
+include LICENSE
+include README.rst
+
+recursive-include tests *
+recursive-exclude * __pycache__
```
Comment (Natim) [inline MANIFEST.in]: This is by default already.

File: kinto_elasticsearch/indexer.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+logger = logging.getLogger(__name__)
+
+
+class Indexer(object):
+    def __init__(self, hosts):
+        self.client = elasticsearch.Elasticsearch(hosts)
+
+    def search(self, bucket_id, collection_id, query, **kwargs):
+        indexname = '%s-%s' % (bucket_id, collection_id)
+        return self.client.search(index=indexname,
```
Comment (Natim) [inline kinto_elasticsearch/indexer.py]: I guess arguments can fit on the same line.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+cornice-swagger==0.5.1
+elasticsearch==5.4.0
+functools32==3.2.3.post2
+hupper==1.0
+iso8601==0.1.11
+jsonpatch==1.15
+jsonpointer==1.10
+jsonschema==2.6.0
+kinto==7.0.1
+## !! Could not determine repository location
```
Comment (Natim) [inline requirements.txt]: We can remove thoose two lines.

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
```
Comment (Natim) [inline setup.py]: We don't need that now that distutils and setuptools have merged.

File: tox.ini [hunk 5]
```diff
@@ -0,0 +1,38 @@
+[tox]
+envlist = py35,kinto-master,flake8
+skip_missing_interpreters = True
+
+[testenv]
+commands = py.test tests --cov-report term-missing --cov kinto_elasticsearch {posargs}
+deps =
+    pytest
```
Comment (Natim) [inline tox.ini]: Can we put thoose on a `dev-requirements.txt` file?

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+language: python
+python: 3.5
```
Comment (Natim) [inline .travis.yml]: Can we also run them for python3.6?

File: Makefile [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+virtualenv: $(PYTHON)
+$(PYTHON):
+	virtualenv $(VENV) --python=python3
+
+tox: $(TOX)
+$(TOX): virtualenv
+	$(VENV)/bin/pip install tox
+
+tests-once: tox
+	$(VENV)/bin/tox -e py35
```
Comment (Natim) [inline Makefile]: Can we use install-dev with dev-requirements and run py.test directly there?

Comment (Natim) [inline Makefile]: See https://github.com/Kinto/kinto/blob/master/Makefile#L90-L91

File: Makefile [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+VENV := $(shell echo $${VIRTUAL_ENV-.venv})
+PYTHON = $(VENV)/bin/python
+TOX = $(VENV)/bin/tox
+TEMPDIR := $(shell mktemp -d)
+
+build-requirements:
+	$(VIRTUALENV) $(TEMPDIR)
+	$(TEMPDIR)/bin/pip install -U pip
+	$(TEMPDIR)/bin/pip install -Ue .
+	$(TEMPDIR)/bin/pip freeze > requirements.txt
```
Comment (Natim) [inline Makefile]: Can we use the improved version of it? See https://github.com/Kinto/kinto/blob/master/Makefile#L64-L68

File: .travis.yml (hunk 6 above)
Comment (leplatrem) [inline .travis.yml]: lots of travis file mess for very few value IMO

File: MANIFEST.in [hunk 9]
```diff
@@ -0,0 +1,6 @@
+include CHANGELOG.rst
+include LICENSE
+include README.rst
+
+recursive-include tests *
+recursive-exclude * *.py[co]
```
Comment (Natim) [inline MANIFEST.in]: This is also automatic.

**Pull request merged**